hello_man.py

```diff
@@ -1,3 +1,9 @@
 # Эта программа запрашивает имя пользователя и здоровается с ним
 user_name = input("What is your name? ")
-primt("Hello " + user_name)
+
+print("Hello " + user_name)
+
+user_age = int(input("How old are you? "))
+
+print(40 - user_age)

```

Comment:
Нет, это какая-то дичь

```suggestion
print("Years until 40: " + (40 - user_age))
```